README.md

```diff
@@ -102,6 +102,7 @@ Amr Sharaf,
 Richard Shi,
 Kavin Singh,
 Alexey Strokach,
+Aina Fatema Asim Merchant,
```

Comment:
I don't think this is the right spot either (the names are sorted alphabetically by family name)